db/seeds.rb

```diff
@@ -1,8 +1,9 @@
-admin_role = Role.find_or_create_by(name: "admin")
 
 grove = Grove.create!(name: "Aspen")
 Grove.create!(name: "Fir")
 Grove.create!(name: "Pine")
+Role.create!(name: "teacher")
```

Comment:
yes, unless we were mistaken, the new default for teachers should be teacher, not admin. We needed that in the DB to check the dropdown was properly displaying all possible roles